v3/src/components/case-table/case-table.scss

```diff
@@ -294,7 +294,28 @@ $table-body-font-size: 8pt;
         .codap-data-cell {
           background-color: var(--rdg-row-selected-background-color);
         }
+        &.has-selected-item.first-child {
```

Comment:
There's probably a reason this and the following are repeated below starting on 308, but just in case it's an oversight, I note it.

Reply:
Resolved in my code review tweaks commit.